<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pb_choose_file">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>choose file</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="te_file_path">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>180</y>
      <width>331</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="te_srv_ip">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_tips_srv_ip">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ip address of recever:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_tips_file_path">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>file path to send:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_apply">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>380</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>send</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_welcome">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>welcome !</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_error_setting">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>320</y>
      <width>371</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_tips_port">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>250</y>
      <width>67</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>port:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="te_port">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>290</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_logo">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>271</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_tips_guide">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>180</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>click &quot;mode&quot; to select function</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_pause">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>380</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>pause</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="mb_choose_mode">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>581</width>
     <height>22</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menumode">
    <property name="title">
     <string>mode</string>
    </property>
    <addaction name="actionSend_file"/>
    <addaction name="separator"/>
    <addaction name="actionReceive_file"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menumode"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSend_file">
   <property name="text">
    <string>Send file</string>
   </property>
  </action>
  <action name="actionReceive_file">
   <property name="text">
    <string>Receive file</string>
   </property>
  </action>
  <action name="actionport">
   <property name="text">
    <string>port</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
